jump-leveleditor/src/main/java/com/bitdecay/jump/leveleditor/render/mouse/DropObjectMode.java

```diff
@@ -31,7 +31,8 @@ public void setObject(Class<? extends RenderableLevelObject> objectClass) {
 
     private void getNewReference() {
```

Comment:
this function existed (and this might be what you are doing) to get a new reference every time we drop an object. This is because LevelObjects get uuid's assigned to them (used for triggers).